spec/factories/provider_enrichments.rb

```diff
@@ -16,12 +16,8 @@
 
 FactoryBot.define do
   factory :provider_enrichment do
-    transient do
-      age { nil }
-    end
-
+    association :provider
```

Comment:
maybe in `after(:build)`?


given a provider have a trait to create it `with_enrichment`...

```ruby

    after(:build) do |enrichment, evaluator|
      if enrichment.provider.blank?
```

Reply:
@defong what's the benefit over doing it this way?

@davidgisbey is `association` needed here? Is there a reason not just to use `provider` here?

Reply:
@misaka don't this link from `providers`

```ruby
   trait :with_enrichment do
      enrichments { [build(:provider_enrichment)] }
    end

```

creates two provider

when you 
do either

```ruby

build(:provider, :with_enrichments)
```

```ruby
create(:provider, :with_enrichments)

```

Reply:
Discussed IRL and @defong now accepts that this works.

Reply:
Actually, @davidgisbey, is the `association` bit needed?

Reply:
There is not. I'll standardise the factories to remove the associations.